dev/builddeps-veloxbe.sh

```diff
@@ -92,8 +92,4 @@ cmake -DBUILD_VELOX_BACKEND=ON -DCMAKE_BUILD_TYPE=$BUILD_TYPE \
       -DENABLE_HBM=$ENABLE_HBM -DVELOX_ENABLE_S3=$ENABLE_S3 -DVELOX_ENABLE_HDFS=$ENABLE_HDFS ..
 make -j
 
-cd $GLUTEN_DIR
-mvn clean package -Pbackends-velox -Pspark-3.2 -DskipTests
```

Comment:
Why delete this?

Reply:
Use [buildbundle-veloxbe.sh](https://github.com/oap-project/gluten/blob/66a24e4ba9f898339adbe7fc85a18d3bc5696bb2/dev/buildbundle-veloxbe.sh). `builddeps-veloxbe.sh` is for building dependencies